src/com/serotonin/mango/vo/report/ReportChartCreator.java

```diff
@@ -58,6 +58,10 @@ public class ReportChartCreator {
      */
     private static final int IMAGE_WIDTH = 930;
     private static final int IMAGE_HEIGHT = 400;
+	private static final int IMAGE_HEIGHT_FOR_DATA_POINT_NAME_IN_LEGEND_PIXELS = 20;
```

Comment:
you need to correct these spaces/tabs. 
This parameter should be associated with the legend and not with the height of the entire image. This is a label, not a name, you can imagine the height and width of the label, and on average the name of the point, what would it mean?
IMAGE_POINT_LABEL_HEIGHT_IN_LEGEND_PIXELS